sdf.lisp

```diff
@@ -102,6 +102,124 @@
          #++(aa-misc:save-image "/tmp/font2h.pnm" image :pnm)
          dest)))))
 
+
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;
+;;  MAXRECT packing
+;;
+
+(defmacro %with-rect ((x y &optional (w (gensym)) (h (gensym))) rect &body body)

```

Comment:
i probably wouldn't have put % there, not particularly dangerous and not exporting it is enough to indicate it is internal